packages/web/src/pages/sign-up-page/components/layout.tsx

```diff
@@ -29,17 +30,40 @@ const messages = {
 type PageProps = FlexProps & {
   as?: ComponentType<any>
   centered?: boolean
+  transition?: 'horizontal' | 'vertical'
 }
 
+const AnimatedFlex = animated(Flex)
+
 export const Page = (props: PageProps) => {
-  const { centered, children, as, ...other } = props
+  const { centered, children, as, transition, ...other } = props
   const { isMobile } = useMedia()
 
+  const styles = useSpring({
```

Comment:
This animation stuff looks clean 💯

Reply:
thanks! if you have any thoughts on how to get the translation going the other way when it's a backwards navigation, that would be clutch